packages/boot/test/bootstrapTests/test-walletSurvivesZoeRestart.ts

```diff
@@ -1,53 +1,32 @@
 /** @file Bootstrap test of liquidation across multiple collaterals */
 import { test as anyTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
 
-import { NonNullish } from '@agoric/assert';
 import process from 'process';
 import type { ExecutionContext, TestFn } from 'ava';
-import type { ScheduleNotification } from '@agoric/inter-protocol/src/auction/scheduler.js';
+
 import { BridgeHandler } from '@agoric/vats';
+import { Offers } from '@agoric/inter-protocol/src/clientSupport.js';
 import {
   LiquidationTestContext,
-  likePayouts,
   makeLiquidationTestContext,
-  scale6,
   LiquidationSetup,
 } from './liquidation.ts';
 
 const test = anyTest as TestFn<LiquidationTestContext>;
 
-//#region Product spec
 const setup: LiquidationSetup = {
```

Comment:
why is LiquidationSetup in a test about walletSurvivesZoeRestart?

Reply:
in order to test that the wallets survive, it's using a vaultLiquidation scenario to exercise the vaults.